gnomad_qc/v4/variant_qc/import_variant_qc_vcf.py

```diff
@@ -90,8 +90,8 @@ def import_variant_qc_vcf(
         split_ht = hl.split_multi_hts(unsplit_ht)
 
         split_ht = split_ht.annotate(
-            info=unsplit_ht.info.annotate(
-                **split_info_annotation(unsplit_ht.info, unsplit_ht.a_index)
+            info=split_ht.info.annotate(
```

Comment:
this is the important fix we needed, I remember, right ?

Reply:
yup